### DISCUSSION TITLE
Unable to find any supported Python versions

### DISCUSSION THREAD

**Discussion by larbing** [vercel/community#6301] [Bug Report]
### Page to Investigate

https://vercel.com/larbings-projects/flask-test/BGvNY5pe9Q4dvJJSHVj6nFVvesjU

### Steps to Reproduce

<img width="752" alt="image" src="https://github.com/vercel/community/assets/1294282/f32408d9-fbe7-4059-a00c-fd44d7e02ca8">

**Comment by larbing** [comment]
<img width="767" alt="image" src="https://github.com/vercel/community/assets/1294282/dee7b55e-fb9c-4963-9364-a2c87d7a76de">

**Comment by amyegan** [comment]
Hey @larbing. This problem looks like the same issue from https://github.com/vercel/community/discussions/6287

Please try the workaround posted in https://github.com/vercel/community/discussions/6287#discussioncomment-8932365 and let me know if that works for you 🙏

**Reply by larbing** [reply]
@amyegan Thank you for your reply
，I recreated it using a template, but it still didn’t work.

<img width="504" alt="image" src="https://github.com/vercel/community/assets/1294282/c93fc548-ef6d-4c8c-9e22-3eafce548afa">


<img width="1330" alt="image" src="https://github.com/vercel/community/assets/1294282/011d22b8-7a52-4c94-961b-ab072481711c">
